docs/articles_en/documentation/legacy-features/transition-legacy-conversion-api.rst

```diff
@@ -293,7 +293,7 @@ Here is the guide to transition from legacy model preprocessing to new API prepr
 
                  mo --input_model MODEL_NAME --batch 2 --output_dir OUTPUT_DIR
 
-            - Not available in OVC tool. Please check Python API.
+            - Not available in OVC tool. Switch to the **Python** tab.
```

Comment:
it is not a link to switch. It is a simple text

Reply:
yep, I've tested possible solutions but none of them provide the required functionality, for now I think we'll have to deal with this issue by guiding users using wording